1.data_preprocessing/1.make_pseudobulks.ipynb

```diff
@@ -0,0 +1,927 @@
+{
```

Comment:
<p>putting the GSEID may be useful in the future if you expand to other single-cell experiments, especially if you want to use some type of harness for running all your notebooks. You will likely have different processing steps for each different GSE experiment.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NDavidsonLab/BuDDI_HGSC_schildkraut_analysis/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>